Add Chime Release Notes

peterschwarz opened the pull request.

peterschwarz committed:
Add jekyll cache to gitignore

Signed-off-by: Peter Schwarz <pschwarz@bitwise.io>

peterschwarz committed:
Change bumper release from latest to 1.1.5

Bumper will no longer be considered "latest" after the chime release, so
these links are corrected to the appropriate version number.

Signed-off-by: Peter Schwarz <pschwarz@bitwise.io>

peterschwarz committed:
Add chime release notes

Add the Hyperledger Sawtooth 1.2 (Chime) release notes to the website.

Signed-off-by: Peter Schwarz <pschwarz@bitwise.io>

agunde406 commented on generator/source/release/chime/notes.md:
```diff
@@ -0,0 +1,101 @@
+get started.
+
+This page describes the new and changed features in Hyperledger Sawtooth 1.2
+(since release 1.1).  Information is organized in the follwing categories:
```
follwing -> following

peterschwarz replied:
Fixed

danintel commented on generator/source/release/chime/notes.md:
```diff
@@ -0,0 +1,101 @@
+<a href="top"></a>
+
+Hyperledger Sawtooth 1.2 (Chime) is now available.  See the [latest
+documentation](https://sawtooth.hyperledger.org/docs/core/releases/1.2.3/) to
```
Hard coding release `1.2.3` to `latest` may be a problem when future point releases are made, such as `1.2.4`. Just something to think about.

peterschwarz replied:
I have proposed internally that we create "release label" symlinks for bumper, chime, and so on.  We'll re-address these then.

annechenette commented on generator/source/release/chime/notes.md:
```diff
@@ -0,0 +1,101 @@
+- Sawtooth PoET is designed for larger networks at the expense of slower /
+  probabilistic finality.
+- Sawtooth Raft is designed for small to medium networks but performs more
+  slowly than Sawtooth PBFT and lacks byzantine fault tolerance.
```
Please indent these three bullets under the first "Support for Sawtooth PBFT..." bullet.  (Add two spaces at the front of each line.)

peterschwarz replied:
Fixed

annechenette commented on generator/source/release/chime/notes.md:
```diff
@@ -0,0 +1,101 @@
+  - resolve a security defect where a correctly admitted validator could game
+    its wait times. Thanks to Huibo Wang, Guoxing Chen, Yinqian Zhang,
+    Zhiqiang Lin for analyzing the poet implementation and identifying the defect.
+  - support the updated attestation service API
```
Please fix the wording and capitalization thusly:

> - PoET has the following patches:
>    - Resolve a security defect ...
>       ...
>    - Support the updated ...

peterschwarz replied:
Fixed

annechenette reviewed (changes requested):
Requested two corrections to the "new" content in the Chime release notes (since I last reviewed the draft release notes).

annechenette commented on generator/source/release/chime/notes.md:
```diff
@@ -0,0 +1,102 @@
+- PoET has the following patches:
+  - Resolve a security defect where a correctly admitted validator could game
+    its wait times. Thanks to Huibo Wang, Guoxing Chen, Yinqian Zhang,
+    Zhiqiang Lin for analyzing the poet implementation and identifying the defect.
```
and Zhiqiang Lin

peterschwarz replied:
Fixed

annechenette reviewed (approved):
Found one small typo (sorry - I missed it in my first pass).  I'll go ahead and approve this PR anyway.

vaporos commented on generator/source/release/bumper/notes.md:
```diff
@@ -12,8 +12,8 @@ release-name: bumper
-Hyperledger Sawtooth 1.1 (Bumper) is now available. See the [latest
-documentation](https://sawtooth.hyperledger.org/docs/core/releases/latest/) to
+Hyperledger Sawtooth 1.1 (Bumper) is now available. See the
+[documentation](https://sawtooth.hyperledger.org/docs/core/releases/1.1.5/) to
```
Shouldn't this be releases/1.1/? (w/link on the filesystem)

peterschwarz replied:
See my above comment to @danintel

rberg2 committed:
Publish chain 1.2.3 docs

This change adds the Sawtooth 1.2.3 docs to the website and the version
selector.

Signed-off-by: Richard Berg <rberg@bitwise.io>
